.github/pull_request_template.md

```diff
@@ -1,2 +1 @@
-- [ ] ** Backport reason (please explain below if this patch should be backported or not) **
-
+**Please replace this line by justification for the backport/\* labels put on for this PR**
```

Comment:
Sorry for the typo in my original suggestion, but "on for" was a typo - should probably be just "on".

Reply:
Changed a bit, I think it makes more sense ,